node_modules/oae-authentication/tests/test-auth-local.js

```diff
@@ -151,17 +148,19 @@ describe('Authentication', function() {
                     var err = err1 || err2;
                     assert.ok(err);
                     assert.equal(err.code, 400);
-                    callback();
+                    return callback();
                 }
             };
 
             // Create a test user
             var userId = TestsUtil.generateTestUserId();
-            RestAPI.User.createUser(camAdminRestContext, userId, 'password', 'Test User', null, function(err, createdUser) {
+            var email1 = TestsUtil.generateTestEmailAddress();
+            var email2 = TestsUtil.generateTestEmailAddress();
+            RestAPI.User.createUser(camAdminRestContext, userId, 'password', 'Test User', {'email': email1}, function(err, createdUser) {
```

Comment:
Perhaps, but this PR is already so big, I'd like to keep that refactor separate so review can still happen (somehow)